### PR TITLE
Add Weave Net as an option for ARM installs

### PR DESCRIPTION
**Pull request by bboreham**
Minimal edits to reflect the current state of support.

I have requested (https://github.com/coreos/flannel/issues/663) that Flannel include the detailed instructions in their docs so this page would just link to them, to cut some clutter.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3250)

### CODE REVIEW

**Comment by bboreham** [comment]
I may have duplicated #2982 ... how come that is merged but we don't see the effect?

**Comment by rade** [comment]
The change got lost in 1c902d0fded05fca11f21cab9db0460be27d0652. Looking at the two parents of that commit, one of them has the changes from #2982 and the other doesn't.

**Comment by rade** [comment]
Presumably there is a high probability that some other changes got lost in that merge too.

**Comment by rade** [comment]
The commit that actually lost the change was d4349f10cba4eb8898d5a4b1339c5a018da533c3. The commit message indicates that there was a conflict in kubeadm.md. It appears that the conflict was resolved by ignoring the changes that had happened in master. cc @chenopis

File: docs/getting-started-guides/kubeadm.md [hunk 1]
```diff
@@ -473,8 +473,10 @@ proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/mu
 deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out
 if there's interest).
 
-Currently, only the pod network flannel is working on multiple architectures.
-You can install it this way:
+Currently, only the pod networks flannel and Weave Net work on multiple architectures.
```
Comment (luxas) [inline docs/getting-started-guides/kubeadm.md]: Now when we have two options, I'm fine with just saying that "these two providers support ARM, follow their guides for more information"

Comment (bboreham) [inline docs/getting-started-guides/kubeadm.md]: Do you have a link for a guide for Flannel on arm?  Or are you saying to put that regardless, so they'll need to provide it?

**Comment by chenopis** [comment]
@rade Sorry, that was totally my mistake. I can reimplement #2982, or should we just handle that in this PR?

**Comment by rade** [comment]
@chenopis I am fine with going with this PR but am concerned that other changes on master may have gotten lost in a similar way by the series of merges from master to release-1.6. If you are sure this was a one-off then fine; otherwise it's probably worth re-examining all the master-to-release-1.6 merges that resulted in a conflict.

**Comment by chenopis** [comment]
@rade To be certain, I will review all the merge conflicts from `master` to `release-1.6`.  Nevertheless, we can still proceed w/ this PR.

**Comment by chenopis** [comment]
@bboreham @luxas @rade Anything else you want in this PR? /lgtm Ping me if it's ready to be merged.

**Comment by bboreham** [comment]
@chenopis from my point of view it can be merged. 

@luxas please can you clarify what you want done

**Comment by chenopis** [comment]
@bboreham Ok, great. @luxas We still need a Tech LGTM on this. Any other changes you would like to see?

**Comment by bboreham** [comment]
@luxas do you want this PR changed?

**Comment by chenopis** [comment]
@rade I caught a couple more changes that were lost in the merge conflicts from 1.6 and restored them in PR #3474. Cheers.

**Pull request merged**